robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAssetManager.java

```diff
@@ -511,15 +511,8 @@ private TypedResource getAndResolve(int resId, String qualifiers, boolean resolv
       value = resolve(value, qualifiers, resId);
     }
 
-    // todo: make the drawable loader put stuff into the normal spot...
-    String resourceTypeName = getResourceTypeName(resId);
-    if (value == null && DrawableResourceLoader.isStillHandledHere(resourceTypeName)) {
-      FileTypedResource typedResource = (FileTypedResource) resourceLoader.getValue(resId, qualifiers);
-      return new TypedResource<>(typedResource.getFsFile(), ResType.FILE, typedResource.getXmlContext());
-    }
-
     // todo: gross. this is so resources.getString(R.layout.foo) works for ABS.
-    if (value == null && "layout".equals(resourceTypeName)) {
+    if (value == null && "layout".equals(getResourceTypeName(resId))) {
```

Comment:
I've never see that exception thrown and after all the resources fixes it feels like we can remove it.